### PR TITLE
firmware_status.py - add missing config var test

### PR DESCRIPTION
**Pull request by crpb**
#28 

> @crpb sorry, my merge was some kind of hasty, can you please fix the condition?

can't reopen merged requests :P.

### CODE REVIEW

**Comment by zerwes** [comment]
> can't reopen merged requests :P.
yes, sorry, as I wrote: my merge was hasty
a new PR is OK

File: files/firmware_status.py [hunk 1]
```diff
@@ -104,6 +108,8 @@
 
 jverlist = json.loads(pr.stdout)
 for verd in jverlist:
+    if verd['version'].endswith('b'):
```
Comment (zerwes) [inline files/firmware_status.py:111]: can you please fix the condition and consider the cfg setting here?
```diff
-    if verd['version'].endswith('b'):
+    if ignore_beta and verd['version'].endswith('b'):
```

**Review by zerwes — CHANGES_REQUESTED** [review]

File: files/firmware_status.py (hunk 1 above)
Comment (crpb) [reply files/firmware_status.py]: every time i use `gh` i confuse myself :see_no_evil:

**Review by zerwes — APPROVED** [review]
LGTM
THX

**Pull request merged**